integrations/maya/src/main/java/org/dash/wallet/integrations/maya/api/ExchangeRateApi.kt

```diff
@@ -17,21 +17,36 @@
 
 package org.dash.wallet.integrations.maya.api
 
+import org.dash.wallet.integrations.maya.model.CurrencyBeaconResponse
 import org.dash.wallet.integrations.maya.model.ExchangeRateResponse
+import org.dash.wallet.integrations.maya.model.FreeCurrencyResponse
 import retrofit2.Response
 import retrofit2.http.GET
+import retrofit2.http.Header
 import retrofit2.http.Query
 
 /**
- * https://exchangerate.host/#/docs
+ * https://v6.exchangerate-api.com/v6/{api-key}}/latest/USD
  */
 interface ExchangeRateApi {
-    @GET("latest")
-    suspend fun getRates(@Query("base") baseCurrencyCode: String): Response<ExchangeRateResponse>
+    @GET("latest/USD")
+    suspend fun getRates(): Response<ExchangeRateResponse>
+}
 
+interface CurrencyBeaconApi {
     @GET("latest")
-    suspend fun getRate(
+    suspend fun getRates(
         @Query("base") baseCurrencyCode: String,
-        @Query("symbols") resultCurrencyCode: String
-    ): Response<ExchangeRateResponse>
+        @Query("symbols") resultCurrencyCode: String,
+        @Query("api_key") apiKey: String = "1xsN7q7S2Lo3gzlmtrpdmLufO96OBlRK"
```

Comment:
This key will be temporary for testing.  We will need a permanent secret key for production.